src/lib.rs

```diff
@@ -65,6 +65,7 @@ pub use self::report::{Report, ReportBuilder, UnresolvedReport};
 #[cfg(feature = "flamegraph")]
 pub use inferno::flamegraph;
 
+#[allow(clippy::all)]
```

Comment:
Why? What problem it's trying to solve?

Reply:
See https://github.com/tikv/pprof-rs/runs/7899854996?check_suite_focus=true, it ignores the clippy error produced in the codes generated by prost.

Reply:
Is it better to be more conservative about suppressing warnings like in https://github.com/tokio-rs/prost/issues/661#issuecomment-1156606409?

Reply:
> Is it better to be more conservative about suppressing warnings like in https://github.com/tokio-rs/prost/issues/661#issuecomment-1156606409?

@sticnarf  I don't think so. If more clippy warnings appears in the prost mod, what we can do is still ignoring them. It seems that being more conservative doesn't make it better.

Reply:
The advantage I can think of is when it's a warning about correctness or performance, it's possible to find and report to the prost upstream. Well, of course, we can rely on others using prost to do the same.

I think both are ok.